redux/ducks/diary.js

```diff
@@ -0,0 +1,59 @@
+import createReducer from 'type-to-reducer';
+
+import { LOADING, SUCCESS, ERROR } from 'constants/redux';
+import api from 'constants/api';
+import request from 'utils/request';
+
+const duck = 'specials/diary';
+
+// constants
+const GET_BY_DAY = `${duck}/GET_BY_DAY`;
+
+const initialState = {
+  pending: false,
+  error: false,
+  data: null,
+};
+
+export default createReducer(
```

Comment:
For reduce boilerplate in reducers I introduce [some utils](https://github.com/babajka/babajka-frontend/pull/15/files#diff-3087a1f10201c2966423b14d9250bcd0) in #16. If you want, we can merge it for sharing.